#ifndef PROTOINC_H_
#define PROTOINC_H_

#include "User.pb.h"
#include "ErrorRet.pb.h"
#include "ProtoNotify.pb.h"
#include "ProtoActivity.pb.h"
#include "ProtoPush.pb.h"
#include "DataCommon.pb.h"
#include "ProtoGM.pb.h"
#include "ProtoBattleDemo.pb.h"
#include "ProtoHero.pb.h"
#include "ProtoBattle.pb.h"
#include "ProtoFriend.pb.h"
#include "ProtoMartialClub.pb.h"
#include "ProtoChampionship.pb.h"
#include "ProtoCards.pb.h"
#include "ProtoBattleBase.pb.h"

#endif /* PROTOINC_H_ */
